src/pages/costModels/costModel/updateMarkupDialog.tsx

```diff
@@ -62,76 +91,111 @@ class UpdateMarkupModelBase extends React.Component<Props, State> {
                 source_uuids: current.sources.map(provider => provider.uuid),
                 source_type: current.source_type === 'OpenShift Container Platform' ? 'OCP' : 'AWS',
                 markup: {
-                  value: this.state.markup,
+                  value: this.state.sign === '-' ? this.state.sign + this.state.markup : this.state.markup,
                   unit: 'percent',
                 },
               };
               updateCostModel(current.uuid, newState, 'updateMarkup');
             }}
             isDisabled={
               isNaN(Number(this.state.markup)) ||
-              Number(this.state.markup) === Number(current.markup.value || 0) ||
+              (Number(this.state.markup) ===
+                Number(
+                  current.markup.value.startsWith('-') ? current.markup.value.substring(1) : current.markup.value || 0
+                ) &&
+                this.state.sign === this.state.origSign) ||
               isLoading
             }
           >
-            {t('cost_models_details.add_rate_modal.save')}
+            {t('save')}
           </Button>,
           <Button
             key="cancel"
             variant="link"
             onClick={() => onClose({ name: 'updateMarkup', isOpen: false })}
             isDisabled={isLoading}
           >
-            {t('cost_models_details.add_rate_modal.cancel')}
+            {t('cancel')}
           </Button>,
         ]}
       >
         <Stack hasGutter>
           <StackItem>{error && <Alert variant="danger" title={`${error}`} />}</StackItem>
-          <StackItem>
-            <Title headingLevel="h2" size="md">
-              {t('cost_models_details.table.columns.name')}
-            </Title>
-          </StackItem>
           <StackItem>
             <TextContent>
-              <Text component={TextVariants.h6}>{current.name}</Text>
+              <Text style={styles.cardDescription}>
+                {t('cost_models_details.description_markup_or_discount_model')}
+              </Text>
```

Comment:
This should probably be a `Title` because it defines a new section, like `Examples`

Reply:
I guess I am a bit confused on this comment. This is the description of the modal/card. The title is above the description on line 81.

`title={t('cost_models_details.edit_markup_or_discount')}`

Reply:
Sorry, I thought this was the "Markup or Discount" title.

Reply:
It confused me because there is another description edit in an unrelated card.